### DISCUSSION TITLE
How to use polling in unmounted component?

### DISCUSSION THREAD

**Discussion by baeharam** [TanStack/query#5356] [Q&A] [answered]
I found that internal implementation of `refetchIntervalInBackground` is `setInterval` so I thought it will work even component is unmounted. But it doesn't work. Is it impossible to implement polling in unmount component?

[Scenario]

1. Mutation is succeeded and Query starts.
2. `refetchInterval` and `refetchIntervalInBackground` is enabled in that query so polling works.
3. Component is unmounted but polling is not works (Cannot see in network tab of Chrome)

**Comment by TkDodo** [comment, marked as answer]
`refetchInterval` is tied to an observer, and if you have no mounted component that calls `useQuery`, you have no observer.
